superset/app.py

```diff
@@ -429,8 +430,19 @@ def init_views(self) -> None:
             )
             appbuilder.add_view_no_menu(AlertLogModelView)
             appbuilder.add_view_no_menu(AlertObservationModelView)
-            if feature_flag_manager.is_feature_enabled("SIP_34_ALERTS_UI"):
-                appbuilder.add_view_no_menu(AlertReportModelView)
+
+        if feature_flag_manager.is_feature_enabled(
+            "ALERT_REPORTS"
+        ) and feature_flag_manager.is_feature_enabled("SIP_34_ALERTS_UI"):
```

Comment:
we recently removed `SIP_34_ALERTS_UI`, `ALERT_REPORTS` should be the only one going forward.

Reply:
https://github.com/apache/incubator-superset/pull/12085 removed it, but looks like there's still a reference in config.py

Reply:
ok, all instances of `SIP_34_ALERTS_UI` are gone